src/Models/User.php

```diff
@@ -3,8 +3,8 @@
 namespace TCG\Voyager\Models;
 
 use Carbon\Carbon;
-use Illuminate\Foundation\Auth\User as Authenticatable;
 use TCG\Voyager\Contracts\User as UserContract;
+use TCG\Voyager\Traits\Authenticatable;
```

Comment:
Same comment, Authenticatable is a class, not a trait.

Reply:
Done